strapi/strapi
# Bug del enter

### What does it do?

I added an onKeyDown functionality to the form to enable Enter key recognition in the EditFolderDialog.tsx file.

### Why is it needed?

 When creating a new folder in the media library, hitting enter after typing the folder name cancels and closes the modal, folder is NOT created.

### How to test it?

I've fixed the bug where creating a new folder in the media library would cancel and close the modal when pressing Enter after typing the folder name; the folder was NOT being created, and now it does. To check the fix, go to the Media Library, click on "add new folder," enter a folder name, and press Enter.


Provide information about the environment and the path to verify the behaviour.

### Related issue(s)/PR(s)

Issue #22352
